lib/chiketto.rb

```diff
@@ -1,9 +1,9 @@
 module Chiketto
   ENDPOINT = 'https://www.eventbriteapi.com/v3/'
-  LEGACY_ENDPOINT = 'https://www.eventbrite.com/json/'
 
   require 'hash_ext'
 
+  require 'chiketto/exception'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.